src/presence_analyzer/tests.py

```diff
@@ -51,7 +52,89 @@ def test_api_users(self):
         self.assertEqual(resp.content_type, 'application/json')
         data = json.loads(resp.data)
         self.assertEqual(len(data), 2)
-        self.assertDictEqual(data[0], {u'user_id': 10, u'name': u'User 10'})
+        self.assertDictEqual(data[0], {'user_id': 10, 'name': 'User 10'})
+
+    def test_mean_time_weekday(self):
+        """
+        Test mean presence time for given user.
+        """
+        resp = self.client.get('/api/v1/mean_time_weekday/12')
```

Comment:
0, if we add more data then probably its gona be next id, more save is checking id = 0